src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -15,6 +15,7 @@ import {
 import ChatItemAttachment from './ChatItemAttachment';
 import {LinkableText} from '../../LinkableText';
 import dimen from '../../../utils/dimen';
+import {LinkableText} from '../../LinkableText';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a redundant import statement for `LinkableText` in your code. You should remove the duplicate to keep the code clean and maintainable.
```diff
- import {LinkableText} from '../../LinkableText';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->